output/slide-deck.qmd

```diff
@@ -530,62 +428,85 @@ git config --list --global
 Paste in terminal with `Shift-Insert`.
```

Comment:
This is Windows-only - looked into it, no Mac equivalent. Maybe just add "in Windows"

Reply:
May it be `COMMAND + V` (https://support.apple.com/en-gb/guide/terminal/trmlshtcts/mac)

Do/Will you use Mac btw?